week-3/E-array-map/exercise.js

```diff
@@ -1,5 +1,19 @@
 // Using the .map() method, create a new array with `numbers` multiplied by 100
 // Write multiple solutions using different syntax (as shown in the README)
-
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
+var multipliedBy100 = numbers.map(function (number) {
+    return number * 100;
+});
+console.log(multipliedBy100);
```

Comment:
This is correct! Well done.

The exercise asks you to write multiple solutions using different syntax. Can you try that?